recipes/pyagg/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.0.1" %}
+{% set sha256 = "30a1c5519696ff868104cfe7d33c8ec52e8698a8be6e21a3b6eb2d0c168f28aa" %}
+
+package:
+  name: pyagg
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/jwiggins/pyagg/archive/{{ version }}.tar.gz
```

Comment:
Abusing? No. But we always try to point to the canonical source code. It is important to know why you are using the fork and, if it is just for a small code change, we could patch it instead and still point to the original source.

PS: Feel free to keep using the fork until your recipe is OK. But it would be nice to change that before we merge this.